File: project/API/MyContextApp/MyContextApp/Models/BuyerDto.cs
using System;

namespace MyContextApp.Models
{
    public class BuyerDto
    {
        public long Id { get; set; }
        public Guid UserId { get; set; }
        public string ReferenceNo { get; set; }
        public string Description { get; set; }
        public string ChainId { get; set; }
        public string Username { get; set; }
        public string Email
        {
            get; set;
        }
    }
}

File: project/API/MyContextApp/MyContextApp/ClientApp/src/app/models/buyer-dto.ts
interface BuyerDto {
  id: number;
  userId: string;
  referenceNo: string;
  chainId: string;
  description: string;
  username: string;
  email: string;
}
interface AssetDto {
  id: number;  
  referenceNo: string;
  recorderId: number;
  patientId: number;
  description: string;
}
interface RecorderDto {
  id: number;
  userId: string;
  referenceNo: string;
  chainId: string;
  description: string;
  username: string;
  email: string;
}

